NEWS.md

```diff
@@ -1,5 +1,7 @@
 # gargle (development version)
 
+* In vignettes, convert all `{r eval=FALSE}` chunks to plain `r` chunks to prevent any chance of vignette code evaluation while maintaining R syntax highlighting (#301, @tanho63)
```

Comment:
```suggestion
* In the non-interactive-auth vignette, convert all `{r eval=FALSE}` chunks to plain `r` chunks to prevent any chance of vignette code evaluation while maintaining R syntax highlighting (#301, @tanho63)
```

Reply:
TBH this bullet won't survive to a release, because we don't mention internal changes like this in NEWS.